hat/assets/js/apps/Iaso/components/tables/ColumnSelectDrawer/index.tsx

```diff
@@ -130,7 +130,7 @@ export const ColumnsSelectDrawer: FunctionComponent<Props> = ({
             {(!disabled || (disabled && !disabledMessage)) && (
                 <Button
                     disabled={disabled}
-                    variant="contained"
+                    variant="outlined"
```

Comment:
Why the switch to outlined? If it's a design/style decision, it should be documented somewhere and we should have a ticket to update the style everywhere (incl. polio)

Reply:
indeed that was a remark from Martin. Search is the primary action here, this is mainly to force user to focus on the search button because search is not enabled by default. I can create a ticket for that, but the purpose here is mainly on those pages where the search is different

Reply:
I change it to be coherent on orgunit page too, for me the rest can stay like that for now